website/static/js/nodeControl.js

```diff
@@ -221,6 +221,7 @@ var ProjectViewModel = function(data) {
      * Add project to the Project Organizer.
      */
     self.addToDashboard = function() {
+        $('#addDashboardFolder').tooltip('hide');
```

Comment:
Instead of doing this whenever we have a button with a tooltip, I wonder if we can generalize this by modifying the tooltip binding handler. Something like:

```js
// osfHelpers.js

ko.bindingHandlers.tooltip = {
    init: function(elem, valueAccessor) {
        $(elem).tooltip(valueAccessor());
        $(elem).click(function() {$(this).tooltip('hide');});
    }
};
```